.github/workflows/test.yml

```diff
@@ -61,11 +53,11 @@ jobs:
             composer-${{ runner.os }}-
             composer-
 
-      - name: "Install tools with phive"
-        run: make tools-install
+      - name: "Install phpunit"
+        run: composer require phpunit/phpunit --dev
```

Comment:
Can we pin it here to some version that we know works? It would be unfortunate if the pipeline breaks just because a new phpunit tag has been released.

Reply:
good point, fixed.